src/get-changed-files.ts

```diff
@@ -151,8 +163,10 @@ export async function getChangedFiles({
   ).filter(Boolean)
 
   log.info({ matchedChangedExistingDirectories })
+  const files = [...matchingFiles, ...matchedChangedExistingDirectories]
 
   return {
-    files: [...matchingFiles, ...matchedChangedExistingDirectories],
+    files,
+    empty: isEmptyArray(files),
```

Comment:
Do we really need separate function with typecheck?

```suggestion
    empty: files.length === 0,
```

Reply:
Yes, we can use this method as well, since we know the data type in the array in advance